rest_framework/filters.py

```diff
@@ -100,13 +101,17 @@ def filter_queryset(self, request, queryset, view):
 
         orm_lookups = [self.construct_search(six.text_type(search_field))
                        for search_field in search_fields]
-
+        and_queries = []
```

Comment:
Not sure I understand the motivation here - feels more obscure after this change rather than more clear.